app/routes/events.js

```diff
@@ -1,11 +1,13 @@
 const router = require('express').Router();
 const bodyParser = require('body-parser');
+const validate = require('express-validation');
 
 const { searchEvents, createEvent, deleteEvent } = require('../controllers/events');
+const { createEventSchema } = require('../validation/events');
```

Comment:
`eventSchema` is more convieint in my own opinion

Reply:
[Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/14/commits/29c56039bf17c26383f823e84cbe5d35d3a8eb8f#diff-21fcb63ee98c4e9d3ba6a929c055da48R3)